xbmc/windowing/osx/WinSystemOSX.mm

```diff
@@ -687,9 +687,10 @@ static void DisplayReconfigured(CGDirectDisplayID display,
 
 bool CWinSystemOSX::CreateNewWindow(const std::string& name, bool fullScreen, RESOLUTION_INFO& res, PHANDLE_EVENT_FUNC userFunction)
```

Comment:
Thanks for looking into it. Thought that might be the case. 
Maybe a comment explaining why this is not needed for OSX would be nice?
Otherwise ignore this.